crates/libcgroups/src/common.rs

```diff
@@ -420,7 +420,7 @@ pub(crate) fn delete_with_retry<P: AsRef<Path>, L: Into<Option<Duration>>>(
         }
 
         std::thread::sleep(delay);
-        attempts += attempts;
+        attempts += 1;
```

Comment:
Hey, maybe setting the initial value 1 would be better than adding 1 to attempts, as the delay in retries is generally exponential, and `attempts += attempts` accounts that exponentiality.
Adding 1 each time would keep a (almost) constant delay between attempts, which is not wrong, but I haven't really seen in context of delays (I have seen them to be exponential)

Reply:
I think parameter `retries` represents the number of retries, so adding one each time is more appropriate.